models/issue_mail.go

```diff
@@ -88,6 +88,10 @@ func mailIssueCommentToParticipants(e Engine, issue *Issue, doer *User, content
 		names = append(names, participants[i].Name)
 	}
 
+	if err := issue.loadRepo(e); err != nil {
```

Comment:
This changes are not in the original PR https://github.com/go-gitea/gitea/pull/5891

Reply:
Yup as per: https://github.com/go-gitea/gitea/pull/5891#issuecomment-458716885

I noticed another nil dereference.

Reply:
Oh, I missed that :)

Reply:
Happy to remove and do another PR if preferred.

Reply:
Ideally, I'd prefer it in another PR since backports are basically cherrypicked.

Reply:
The only trouble is that the PR doesn't then completely fix the bug...

@techknowlogick Should we keep the repo fix in this PR or make another with it separately?

Reply:
Yeah, preferably two PRs for the reasons @adelowo listed above. However in this case I think we can keep it as one PR.